packages/reports/addon/components/filter-builders/number-dimension.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+import BaseComponent from './base';
+import { computed } from '@ember/object';
+import { A as arr } from '@ember/array';
+
+export default class FilterBuildersNumberDimension extends BaseComponent {
+  /**
+   * @property {Object} requestFragment - filter fragment from request model
+   * @type Any
+   */
+  requestFragment = undefined;
+
+  /**
+   * @property {Array} supportedOperators
+   * @override
+   */
+  get supportedOperators() {
+    return [
+      {
+        id: 'gt',
+        name: 'Greater than (>)',
+        valuesComponent: 'filter-values/value-input'
+      },
+      {
+        id: 'gte',
+        name: 'Greater than or equals (>=)',
+        valuesComponent: 'filter-values/value-input'
+      },
+      {
+        id: 'lt',
+        name: 'Less than (<)',
+        valuesComponent: 'filter-values/value-input'
+      },
+      {
+        id: 'lte',
+        name: 'Less than or equals (<=)',
+        valuesComponent: 'filter-values/value-input'
+      },
+      {
+        id: 'eq',
+        name: 'Equals (=)',
+        valuesComponent: 'filter-values/value-input'
+      },
+      {
+        id: 'neq',
+        name: 'Not equals (!=)',
+        valuesComponent: 'filter-values/value-input'
+      },
+      {
+        id: 'bet',
+        name: 'Between (<=>)',
+        valuesComponent: 'filter-values/range-input'
+      },
+      {
+        id: 'nbet',
+        name: 'Not between (!<=>)',
+        valuesComponent: 'filter-values/range-input'
+      }
+    ];
+  }
+
+  /**
+   * @property {Object} filter
+   * @override
+   */
+  @computed('requestFragment.{operator,dimension,values.[]}')
+  get filter() {
+    const { requestFragment } = this;
+    const serializedFilter =
+      typeof requestFragment?.serialize === 'function' ? requestFragment.serialize() : requestFragment;
```

Comment:
Why do we need to serialize the fragment? Do we do this elsewhere?

Reply:
We do this everywhere in the filter builders.

Reply:
hmm, we don't do it here:
https://github.com/yahoo/navi/blob/c56ef587f81b6e098f441397d289a3d2cb4ba096/packages/reports/addon/components/filter-builders/dimension.js#L85-L97

Reply:
We don't do it in date-dimension bulder, which is most similar to this one.

Reply:
testing something...